### PR TITLE
Fix definitions of various AbstractEventLoop methods

### PR DESCRIPTION
**Pull request by telendt**
@sii fixed some of them in aa6f4a0 (#753), this PR fixes the remaining ones.

### CODE REVIEW

File: stdlib/3.4/asyncio/events.pyi [hunk 1]
```diff
@@ -131,12 +131,12 @@ class AbstractEventLoop(metaclass=ABCMeta):
...
     # Signal handling.
     @abstractmethod
-    def add_signal_handler(self, sig: int, callback: Callable[[], Any], *args: List[Any]) -> None: ...
+    def add_signal_handler(self, sig: int, callback: Callable[..., Any], *args: List[Any]) -> None: ...
     @abstractmethod
     def remove_signal_handler(self, sig: int) -> None: ...
     # Error handlers.
     @abstractmethod
-    def set_exception_handler(self, handler: Callable[[], Any]) -> None: ...
+    def set_exception_handler(self, handler: Callable[..., Any]) -> None: ...
```
Comment (telendt) [inline stdlib/3.4/asyncio/events.pyi]: This one is wrong, I'll fix that.

https://docs.python.org/3/library/asyncio-eventloop.html#asyncio.AbstractEventLoop.set_exception_handler
> If handler is a callable object, it should have a matching signature to (loop, context), where loop will be a reference to the active event loop, context will be a dict object (see call_exception_handler() documentation for details about context).

File: stdlib/3.4/asyncio/events.pyi [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import sys
-from typing import Any, Awaitable, TypeVar, List, Callable, Tuple, Union, Dict, Generator, overload, Optional
+from typing import Any, Awaitable, Callable, Dict, Generator, List, Optional, Tuple, TypeVar, Union, overload
```
Comment (telendt) [inline stdlib/3.4/asyncio/events.pyi]: I had to import Dict, so I thought I might as well order it.

File: stdlib/3.4/asyncio/events.pyi [hunk 3]
```diff
@@ -8,6 +8,7 @@ from asyncio.tasks import Task
 __all__ = ...  # type: str
 
 _T = TypeVar('_T')
+_Context = Dict[str, Any]
```
Comment (telendt) [inline stdlib/3.4/asyncio/events.pyi]: https://docs.python.org/3/library/asyncio-eventloop.html#asyncio.AbstractEventLoop.call_exception_handler

> context is a dict object containing the following keys [...]

**Comment by gvanrossum** [comment]
Thanks! You hit the nail ight on the head.

**Pull request merged**